substrate/bin/node/runtime/src/lib.rs

```diff
@@ -3453,6 +3453,7 @@ impl_runtime_apis! {
 				code,
 				data,
 				salt,
+				NonceAlreadyIncremented::Yes,
```

Comment:
This is a dry-run. Only transactions do increment.

```suggestion
				NonceAlreadyIncremented::No,
```